test.py

```diff
@@ -26,9 +26,11 @@ def face_recognized(ctx):
 
 
 registry.register(name="hi", states=(hello_world, generic_answer, face_recognized))
+verbaliser.add_folder('/home/dolp/git/ravestate/resources/phrase_lists/')
```

Comment:
Does this work on all devices?

Reply:
Oh, forgot to remove it.
Used it for testing, thanks for the heads up